.github/workflows/release-please.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow opens and updates a pull request with a new package version
+# based on code changes.
```

Comment:
Could you expand those comments in a similar manner as https://github.com/ButterCMS/buttercms-ruby/pull/28/files#diff-2c84033033d49186c63e6adcd705f63b11ae6814cd76c152c9c486d389fbccf3R4-R7?

Reply:
@Toreno96 Resolved in https://github.com/ButterCMS/buttercms-csharp/pull/40/commits/f2e4a865ea97785f7146bd83c4808969a15a6fd4